deploy/helm/clickhouse-operator/templates/generated/Service-clickhouse-operator-metrics.yaml

```diff
@@ -18,7 +18,7 @@ metadata:
 spec:
   ports:
     - port: 8888
-      name: clickhouse-metrics
+      name: ch-metrics
     - port: 9999
-      name: operator-metrics
+      name: op-metrics
```

Comment:
Seems like port names no longer match the contents of a service monitor.

Reply:
@weisdd thanks for suggestion https://github.com/Altinity/clickhouse-operator/pull/1788